src/screens/HomeScreen.tsx

```diff
@@ -37,6 +65,44 @@ export default function HomeScreen() {
   return (
     <ViewContainer>
       <MapView style={styles.map} region={DEFAULT_LOCATION}>
+        <Polygon
+          coordinates={[
+            {
+              latitude: 37.732937,
+              longitude: -122.42002,
+            },
+            {
+              latitude: 37.733048,
+              longitude: -122.420035,
+            },
+            {
+              latitude: 37.733098,
+              longitude: -122.420051,
+            },
+            {
+              latitude: 37.733076,
+              longitude: -122.420177,
+            },
+            {
+              latitude: 37.732894,
+              longitude: -122.420154,
+            },
+            {
+              latitude: 37.732937,
+              longitude: -122.42002,
+            },
+          ]}
+          fillColor="rgba(0,0,255,0.5)"
+          strokeColor="rgba(0,0,255,0.5)"
+          strokeWidth={2}
+        >
+          <Callout>
+            <Text> Fruit Ally </Text>
```

Comment:
whats the callout / text for?

Reply:
I saw in a video where they used <Callout> then <Text> to create labels for polygons. When I tried it here it did not work, should I delete this completely? It was more of an experimental thing and I know you said that PRs should be complete and not have extra stuff